app/models/job/Job.scala

```diff
@@ -72,7 +72,7 @@ case class Job(
           }
         case "export_tiff" =>
           Some(s"$dataStorePublicUrl/data/exports/${_id.id}/download")
-        case "infer_nuclei" | "infer_neurons" =>
+        case "infer_nuclei" | "infer_neurons" | "apply_merger_mode" =>
```

Comment:
I am not sure, whether this is everything I need to do here for the new job. I did not test the jobs list view and therefore this is likely untested.